examples/example_modelvis.ipynb

```diff
@@ -0,0 +1,275 @@
+    "    return clean"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9f5e15d0-a54a-403c-9130-b83634f3fc15",
+   "metadata": {},
+   "source": [
+    "### Setup\n",
+    "Initialize the Node Editor instance 'ne' and load a model into it.\n",
+    "\n",
+    "Use the filepath *sample_test_models/model_trifecta.yml* for a sample model.\n",
+    "\n",
+    "As a note, the Trifecta sample model consists of three models\n",
+    "1. Model RNG - A model which outputs a random number 4 times (1 per line in the input text file)\n",
+    "2. Model Reader - Reads said input text file line by line\n",
+    "3. Model ROTXX - Rotates each line by the number generated and outputs it encoding in a ROT-XX format (think of ROT13 encoding, where each letter in the alphabet is moved 13 places, but here the number is random)\n",
+    "\n",
+    "**Make sure you manually right click and add all of your desired components once the editor appears.**\n",
+    "\n",
+    "As of March 30th, 2022, there is a bug that does not allow for the components to be linked. We are working on fixing that but for production it will still execute normally."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8bdc024d-5cf7-4602-bf22-53b382ccf8e4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ne = jlne.NodeEditor()\n",
+    "jlne.load_model(ne)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "162e6c02-5215-45f0-8f8f-245877a87c21",
+   "metadata": {},
+   "source": [
+    "### Callback Classes and Display\n",
+    "This is where things get a bit messy and will need to be adjusted based on your specific model.\n",
+    "\n",
+    "The class name doesn't matter (*CBJLNE* - CallBack JLNE) so long as it is called properly. The comments in the code will explain each portion to better help the flow of this example.\n",
+    "\n",
+    "*args[0]* is where the data in the runner is stored (i.e. the data that is being moved around, in the sample model's case, the number, line of text, and the rotated output)\n",
+    "\n",
+    "*Note: You will see that we use the 3, 4, and 5 indices for the children but this modifies the sixth, seventh, and eighth rows in the tab. This is due to the Inputs/Outputs in JLNE spacing out their actual name to the next line. For example, under Inputs in Tab 3 it says \"Slot 1: input_rotxx (temp_in0)\" in the next row, but this is still all in the 2nd ipywidgets Label, just spread out for visual clarity in case of multiple Inputs/Outputs. If you need more to display, continue the index trend and do not worry about the rows. Be sure to add them to the initial length check as well so all rows are created properly.*"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3efbcbef-887c-462d-af0b-a4257f3eba6c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the callback class and have the third component be updated with live information\n",
```

Comment:
Shouldn't this class be defined in the library?

Reply:
Not necessarily since we have to adjust the functions inside the class for each model, no?